onnxscript/tools/memory_peak.py

```diff
@@ -0,0 +1,243 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
```

Comment:
```suggestion
# Copyright (c) Microsoft Corporation.
# Licensed under the MIT License.
```

Reply:
Looks like some changes applied were not reverted? Not sure if my view was outdated. I would double check

Reply:
Sometimes, it is added by ruff format or isort,  I don't remember. I don't use lintrunner as it does not work for me.

Reply:
That shouldn't have anything to do with the linters. It's just the copyright header